config/jobs/cert-manager/cert-manager/cert-manager-periodics.yaml

```diff
@@ -300,12 +315,11 @@ periodics:
             value: "1"
   - agent: kubernetes
     annotations:
-      description: Runs cert-manager upgrade test every 8 hours
+      description: Runs cert-manager upgrade from latest published release
```

Comment:
note: Generally I've tried to keep descriptions the same, but where they mentioned an interval like here I've removed it. It seems to just duplicate the actual test specification.